pkgs/applications/misc/opencpn/default.nix

```diff
@@ -48,9 +51,32 @@ stdenv.mkDerivation rec {
     hash = "sha256-sNZYf/2gtjRrrGPuazVnKTgcuIQpKPazhexqlK21T4g=";
   };
 
-  nativeBuildInputs = [ cmake lsb-release pkg-config ];
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/OpenCPN/OpenCPN/pull/2473.patch";
+      sha256 = "sha256-Sb4FE9QJA5kMJi52/x1Az6rMTS3WSURPx4QAhcv2j9E=";
+    })
+  ];
+
+  postPatch = ''
+    substituteInPlace cmake/TargetSetup.cmake \
+      --replace '"sw_vers" "-productVersion"' '"echo" "1"'
+    sed -i '/fixup_bundle/d' CMakeLists.txt
```

Comment:
Why is this needed unconditionally now?

Reply:
Since these commands are darwin-only and won't affect Linux builds.

Reply:
Then move them to the portion that's darwin-only, IMO

Reply:
Done.